portfolio/src/main/webapp/index.html

```diff
@@ -7,10 +7,37 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+    <div id="fullpage">
+      <div class="section"></div>
+      <div class="section"></div>
+      <div class="section" id="projects">
+        <a class="card">
```

Comment:
The [`<a>` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) is usually associated with hyperlinks. Here it looks like it's used as a contained for two `<span>` tags and it's missing an `href` attribute. Either add an `href` attribute, or replace this with a more appropriate content tag like a `<div>`, here and below. 
Reference:
https://google.github.io/styleguide/htmlcssguide.html#Semantics